about.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About NTS</title>
+    <script src="https://kit.fontawesome.com/fe8431d094.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://use.typekit.net/ubw2tuq.css">
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+
+  <nav class="upper">
+    <ul class="socials">
+      <li>
+        <a href="">
+          <i class="fa-brands fa-facebook-f"></i>
+        </a>
+      </li>
+      <li>
+        <a href="">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+      </li>
+      <li>english</li>
+      <li>my page</li>
+      <li>logout</li>
+    </ul>
+  </nav>
+
+  <header>
+
+    <main class="links">
+      <nav>
+        <a href="/index.html" class="logo">
+          <img src="images/logo.png" alt="tech hub logo" srcset="">
+        </a>
+        <div class="icon">
+          <i class="fa-sharp fa-solid fa-bars"></i>
+        </div>
+        <ul id="tray">
+          <li>
+            <a href="./index.html">home</a>
+          </li>
+          <li>
+              <a href="/about.html">about</a>
+          </li>
+          <li>
+              <a href="">program</a>
+          </li>
+          <li>
+              <a href="">join</a>
+          </li>
+          <li>
+              <a href="">sponsor</a>
+          </li>
+          <li id="larger">
+              <a href="">CC campaign</a>
+          </li>
+
+          <li class="icon">
+            <i class="fa-sharp fa-solid fa-xmark"></i>
+          </li>
+        </ul>
+      </nav>
+    </main>
+            
+    <main class="intro">
+      <h3>"Hello! you're welcome"</h3>
+      <h1> To annual national tech submit 2022</h1>
+      <div class="big">
```

Comment:
- [ ] As suggested by the previous code reviewer, please make sure you stick to the design as much as possible, by using the right colors, fonts, and backgrounds.

__Hints__ : 
   - font: Lato, 
   - __background image to the h1__
![title](https://user-images.githubusercontent.com/97834160/210814852-0e2ea33f-fef3-4169-b97b-cf9b298c98bb.png)

   - color: ![colors](https://user-images.githubusercontent.com/97834160/210814008-f693f713-bd2d-4341-a7b5-7e12a2f64f0d.png)

As it was mentioned by the previous reviewer [here](https://github.com/Tobby8629/Capstone-project/pull/1#discussion_r1061948798)